src/transformers/models/resnet/configuration_resnet.py

```diff
@@ -75,6 +78,8 @@ class ResNetConfig(PretrainedConfig):
     """
     model_type = "resnet"
     layer_types = ["basic", "bottleneck"]
+    # note that this assumes there are always 4 stages
+    stage_names = ["stem", "stage1", "stage2", "stage3", "stage4"]
```

Comment:
Shouldn't this be generated based the length of the `depths` parameter passed in the init?

Reply:
Thanks for suggesting, added that!